env0/resource_user_environment_assignment.go

```diff
@@ -0,0 +1,118 @@
+package env0
+
+import (
+	"context"
+	"log"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceUserEnvironmentAssignment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceUserEnvironmentAssignmentCreate,
+		UpdateContext: resourceUserEnvironmentAssignmentUpdate,
+		ReadContext:   resourceUserEnvironmentAssignmentRead,
+		DeleteContext: resourceUserEnvironmentAssignmentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"user_id": {
+				Type:        schema.TypeString,
+				Description: `id of the user. Note: can also be an id of a "User" API key`,
```

Comment:
are you sure about the note? haven't tried it with 'user' api-keys

Reply:
I'll remove the comment.